sgv2-graphqlapi/src/main/java/io/stargate/sgv2/graphql/impl/GraphqlServiceStarter.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.sgv2.graphql.impl;
+
+import com.github.rvesse.airline.SingleCommand;
+import com.github.rvesse.airline.annotations.Command;
+import com.github.rvesse.airline.annotations.Option;
+import com.github.rvesse.airline.help.cli.CliCommandUsageGenerator;
+import com.github.rvesse.airline.parser.ParseResult;
+import com.github.rvesse.airline.parser.errors.ParseException;
+import io.stargate.core.metrics.api.HttpMetricsTagProvider;
+import io.stargate.core.metrics.api.NoopHttpMetricsTagProvider;
+import io.stargate.core.metrics.impl.MetricsImpl;
+import java.io.IOError;
+import java.io.IOException;
+
+@Command(name = "sgv2-graphqlapi")
+public class GraphqlServiceStarter {
+
+  @Option(
+      name = {"--disable-playground"},
+      title = "disable-playground",
+      description = "whether to disable the GraphQL Playground UI (exposed at /playground)")
+  private boolean disablePlayground;
+
+  @Option(
+      name = {"--disable-default-keyspace"},
+      title = "disable-playground",
+      description =
+          "whether to disable the \"default keyspace\" feature (which consists in falling back to "
+              + "the oldest keyspace if not specified in the URL path)")
+  private boolean disableDefaultKeyspace;
+
+  public void start() throws Exception {
+    final MetricsImpl metricsImpl = new MetricsImpl();
+    final HttpMetricsTagProvider httpMetricsTags = new NoopHttpMetricsTagProvider();
```

Comment:
we need to find a way to load a correct tag provider here, otherwise metrics will not be matched to current one.. Same problem exists in the rest-api, maybe we should create a dedicated ticket for this..

Reply:
Yes, a separate ticket would be better because I don't know exactly how to approach this.